Src/Sentry.Xamarin/Extensions/SentryXamarinOptionsExtensions.cs

```diff
@@ -38,12 +38,14 @@ public static void DisableOfflineCaching(this SentryXamarinOptions options)
             options.InternalCacheEnabled = false;
         }
 
-        internal static string DefaultCacheDirectoyPath(this SentryXamarinOptions options)
+        internal static string DefaultCacheDirectoyPath(this SentryXamarinOptions _)
             => _internalCacheDefaultPath.Value;
 
         internal static void ConfigureSentryXamarinOptions(this SentryXamarinOptions options)
         {
-            options.Release ??= $"{AppInfo.PackageName}@{AppInfo.VersionString}";
+            options.Release ??= DeviceInfo.Platform != DevicePlatform.UWP ?
+                $"{AppInfo.PackageName}@{AppInfo.VersionString}+{AppInfo.BuildString}" :
+                $"{AppInfo.PackageName}@{AppInfo.VersionString}";
```

Comment:
```suggestion
            options.Release ??= $"{AppInfo.PackageName}@{AppInfo.VersionString}+{AppInfo.BuildString}";
```

Reply:
Idelly we open an issue on getsentry/sentry explaining this (with examples/screenshot) and suggesting support for 1.1.1.1 format